### PR TITLE
New All In one light card 

### PR DESCRIPTION
**Pull request by schumijo**
Discussion #193 
New All In one light card 
Replace old light cards
Legacy templates enabled
Templates splitted in different YAML
Added mkdocs documentation @stokkie90 : can you take a look please ?

I hope to be aligned with all new features (split, doc, etc...) 😉

### CODE REVIEW

File: docs/usage/cards/light.md [hunk 1]
```diff
@@ -2,9 +2,52 @@
...
+- legacy ```card_light_slider_collapse```
+- legacy ```card_light_slider_horizontal```
+- custom ```card_light_color``` by basbruss 
+- custom ```card_light_horizontal_color``` by basbruss 
+- custom ```card_light_slider_color``` by basbruss 
+- custom ```card_light_slider_collapse_color``` by basbruss 
+
+!!! warning
+
+    This card has backward compatibilty with older template except custom names and icons. It means variables like ```ulm_card_XXX_name``` and ```ulm_card_XXX_icon``` must be replaced by ```ulm_card_light_name``` and ```ulm_card_light_icon```.
```
Comment (stokkie90) [inline docs/usage/cards/light.md:22]:
```suggestion
    This card has backward compatibilty with older template except custom names and icons. It means variables like `ulm_card_XXX_name` and `ulm_card_XXX_icon` must be replaced by `ulm_card_light_name` and `ulm_card_light_icon`.
```

File: docs/usage/cards/light.md [hunk 2]
```diff
@@ -2,9 +2,52 @@
...
+- legacy ```card_light_slider_collapse```
+- legacy ```card_light_slider_horizontal```
+- custom ```card_light_color``` by basbruss 
+- custom ```card_light_horizontal_color``` by basbruss 
+- custom ```card_light_slider_color``` by basbruss 
+- custom ```card_light_slider_collapse_color``` by basbruss 
+
+!!! warning
+
+    This card has backward compatibilty with older template except custom names and icons. It means variables like `ulm_card_XXX_name` and `ulm_card_XXX_icon` must be replaced by `ulm_card_light_name` and `ulm_card_light_icon`.
```
Comment (CM000n) [inline docs/usage/cards/light.md:22]: you must remove this indentation.
Perhaps you would like to use here also best a warning admonition?
https://squidfunk.github.io/mkdocs-material/reference/admonitions/

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/legacy_templates/cards.yaml [hunk 3]
```diff
@@ -49,3 +49,39 @@ media_buttons:
...
+  template: "card_light"
+  variables:
+    ulm_card_light_enable_slider: true
+    ulm_card_light_enable_color: false
+card_light_slider_collapse_color:
+  template: "card_light"
+  variables:
+    ulm_card_light_enable_collapse: true
+    ulm_card_light_enable_slider: true
+    ulm_card_light_enable_color: false
```
Comment (CM000n) [inline custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/legacy_templates/cards.yaml:87]: You have a missing end of file line here

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_light.yaml [hunk 4]
```diff
@@ -2,27 +2,251 @@
...
+  show_name: false
+  show_label: false
+  state:
+    - value: "on"
+      styles:
+        card:
+          - background-color: >
+              [[[
+                  var color = entity.attributes.rgb_color;
+                  if (variables.ulm_card_light_enable_background_color == true){
```
Comment (CM000n) [inline custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_light.yaml:26]: Trailing whitespaces at end of line

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_light.yaml [hunk 5]
```diff
@@ -2,27 +2,251 @@
...
+                  } else {
+                    return variables.ulm_unavailable;
+                  }
+                ]]]
+              styles:
+                name:
+                  - color: >-
+                      [[[
+                          var color = entity.attributes.rgb_color;
+                          if (variables.ulm_card_light_enable_background_color == true){
```
Comment (CM000n) [inline custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_light.yaml:168]: Trailing whitespaces at end of line

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_light.yaml [hunk 6]
```diff
@@ -2,27 +2,251 @@
...
+                              }
+                            }
+                          }
+                          return 'rgba(var(--color-yellow-text),1)'
+                      ]]]
+                label:
+                  - color: >-
+                      [[[
+                          var color = entity.attributes.rgb_color;
+                          if (variables.ulm_card_light_enable_background_color == true){
```
Comment (CM000n) [inline custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_light.yaml:191]: Trailing whitespaces at end of line

File: docs/usage/cards/light.md (hunk 2 above)
Comment (CM000n) [reply docs/usage/cards/light.md:22]: nevermind, it is actual a warning admonition that you are using :-D
So ignore that and I will take care on the markdownlinting adjustments later on

**Pull request merged**